mitmproxy/flowfilter.py

```diff
@@ -382,6 +381,40 @@ def __call__(self, f):
         return f.server_conn.address and self.re.search(r)
 
 
+class FReplay(_Action):
+    code = "r"
+    help = "Match replayed flows"
+
+    def __call__(self, f):
+        return f.is_replay is not None
+
+
+class FReplayClient(_Action):
+    code = "rc"
+    help = "Match replayed client flows"
+
+    def __call__(self, f):
+        return f.is_replay == 'request'
+
+
+class FReplayServer(_Action):
+    code = "rs"
+    help = "Match replayed server flows"
+
+    def __call__(self, f):
+        return f.is_replay == 'response'
+
+
+class FMeta(_Rex):
+    code = "meta"
+    help = "Flow metadata"
+    flags = re.MULTILINE
+
+    def __call__(self, f):
+        m = http.Headers(**{key: repr(value) for key, value in f.metadata.items()})
```

Comment:
That's quite an abuse of our headers class here. Can we search on `f"{key}: {value}"` strings instead please?